.travis.yml

```diff
@@ -34,6 +34,13 @@ before_install:
 install:
   - dotnet restore
 
+before_script:
+  - npm install -g eclint
+  - git rm .editorconfig
```

Comment:
If I'm understanding correctly, this is because not doing this would throw errors on other settings. Why not fix those errors at the same time? Duplicating the file patterns to apply the checks to is kind of ugly.

Reply:
> …this is because not doing this would throw errors on other settings.

Correct.

> Why not fix those errors at the same time?

The errors we cannot and _don't want to_ fix have to do with indentation. `eclint` is too strict although I believe you can negate the setting in `.editorconfig` to silence it but it would amount to the same thing as maintaining a separate list here.

> Duplicating the file patterns to apply the checks to is kind of ugly.

I agree but remember that `.editorconfig` has been removed during the checks.

Do you see a better way?